common/scripts/babel-conditional-preprocess.js

```diff
@@ -50,6 +50,10 @@ exports.default = babelHelper.declare((_api, opts) => {
         Handle(path, annotations);
       },
 
+      TSDeclareMethod(path) {
```

Comment:
Need to add this in order to conditionally compile some of the overloaded function and event handlers that the calling sdk is coming out with in 1.4.1-beta.1

Reply:
I'm very in favor of adding these -- it seems we need to add more and more -- @PorterNan is there any harm in adding them? Hypothetically what if we just added loads now would there be any risk?

Reply:
Add away.

The only ones to be wary of are ones that allow conditional compilation directives in the middle of expressions -- those can allow us to write (even more) indecipherable conditional code.

Stuff like methods / classes / statements are all good. This is just an oversight.